src/snyk/service.ts

```diff
@@ -35,35 +35,76 @@ export class SnykService {
       throw new Error("Missing token for Snyk")
     }
 
-    const response = await fetch(
-      `https://snyk.io/api/v1/org/${encodeURIComponent(
-        snykAccountId,
-      )}/projects`,
-      {
+    let backportedProjects: SnykProject[] = []
+    const snykRestApiVersion = "2023-08-04"
+
+    let nextUrl: string | undefined = `/orgs/${encodeURIComponent(
+      snykAccountId,
+    )}/projects?version=${snykRestApiVersion}&meta.latest_dependency_total=true&meta.latest_issue_counts=true&limit=100`
+
+    /* The Snyk REST API only allows us to retrieve 100 projects at a time.
+     * The "links.next" value in the response gives us a pointer to the next 100 results.
+     * We continue calling the Snyk API and retrieving more projects until links.next is null
+     * */
+    while (nextUrl) {
+      const response = await fetch(`https://api.snyk.io/rest${nextUrl}`, {
         method: "GET",
         headers: {
           Accept: "application/json",
           Authorization: `token ${token}`,
         },
         agent: this.config.agent,
-      },
-    )
+      })
 
-    if (response.status === 401) {
-      process.stderr.write("Unauthorized - removing token\n")
-      await this.tokenProvider.markInvalid()
-    }
+      if (response.status === 401) {
+        process.stderr.write("Unauthorized - removing token\n")
+        await this.tokenProvider.markInvalid()
+      }
+
+      if (!response.ok) {
+        throw new Error(
+          `Response from Snyk not OK (${response.status}): ${JSON.stringify(
+            response,
+          )}`,
+        )
+      }
+
+      // Check if the Sunset header is present in the response
+      const sunsetHeader = response.headers.get("Sunset")
```

Comment:
HTTP headers are not case sensitive, so I recommend that we explicitly check for the lowercase variant as well.
```suggestion
      const sunsetHeader = response.headers.get("Sunset") || response.headers.get("sunset")
```